micro_sam/util.py

```diff
@@ -258,6 +285,7 @@ def export_custom_sam_model(
     checkpoint_path: Union[str, os.PathLike],
     model_type: str,
     save_path: Union[str, os.PathLike],
+    device: str = None,
```

Comment:
We don't want to have the device as an argument here: this function serialize a SAM checkpoint from our training to the normal weight format, so that it can be used by the napari tool (or by any other software that uses the SAM library).
This export should always go to the CPU, to ensure the model can be easily loaded.
So please remove the device argument here.

Reply:
I have fixed this already.

Reply:
Ah, ok - thank you!